From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=A1t=C3=A9=20Kocsis?= <kocsismate@woohoolabs.com>
Date: Mon, 21 Sep 2026 20:01:43 +0200
Subject: [PATCH 23/26] URL: Keep replacement file drive paths hierarchical
 (#423)

WHATWG file state (https://url.spec.whatwg.org/#file-state) step 4.4.3.2 resets the path to an empty list. Do not mark it opaque, as that makes subsequent pathname updates silently do nothing.
---
 source/lexbor/url/url.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/source/lexbor/url/url.c b/source/lexbor/url/url.c
index 8187c0f..dcbc410 100644
--- a/source/lexbor/url/url.c
+++ b/source/lexbor/url/url.c
@@ -2086,7 +2086,6 @@ again:
                 }
 
                 lxb_url_path_set_null(url);
-                url->path.opaque = true;
             }
         }
 
